core/src/main/scala/org/bitcoins/core/currency/package.scala

```diff
@@ -42,4 +42,9 @@ package object currency {
     new Ordering[CurrencyUnit] {
       override def compare(x: CurrencyUnit, y: CurrencyUnit): Int = x.compare(y)
     }
+
+  implicit val satoshisOrdering: Ordering[Satoshis] =
```

Comment:
Doesn't `Satoshis extend CurrencyUnit`? Does the above not cover this? :(

Reply:
It gave me errors :(